pkg/controller/disruption/disruption.go

```diff
@@ -595,11 +595,18 @@ func (dc *DisruptionController) trySync(pdb *policy.PodDisruptionBudget) error {
 		dc.recorder.Eventf(pdb, v1.EventTypeNormal, "NoPods", "No matching pods found")
 	}
 
-	expectedCount, desiredHealthy, err := dc.getExpectedPodCount(pdb, pods)
+	expectedCount, desiredHealthy, nakedPods, err := dc.getExpectedPodCount(pdb, pods)
 	if err != nil {
 		dc.recorder.Eventf(pdb, v1.EventTypeWarning, "CalculateExpectedPodCountFailed", "Failed to calculate the number of expected pods: %v", err)
 		return err
 	}
+	// We have naked pods, instead of erroring and hotlooping in disruption controller, enqueue and see if the
+	// error goes away. To identify the pods, we're emitting an event.
+	if len(nakedPods) > 0 {
+		dc.recorder.Eventf(pdb, v1.EventTypeWarning, "NakedPodsFound",
```

Comment:
Maybe `UnmanagedPodsFound` and replace `nakedPods` with `unmanagedPods` across the board.